mira.zsh-theme

```diff
@@ -10,12 +10,12 @@ else
 fi
 
 local current_dir='%{$terminfo[bold]$fg[blue]%}%~ %{$reset_color%}'
-local git_branch='$(git_prompt_info)'
+local vcs_branch='$(git_prompt_info)$(hg_prompt_info)'
```

Comment:
Where does `hg_prompt_info` come from? Is it native to `oh-my-zsh`? I can't find anything related to this from a cursory Google search.

Reply:
Good catch; turns out this comes from a 3rd-party plugin. I documented that above the `local vcs_branch`.

Reply:
Sounds good!